server/src/projects/projects.service.ts

```diff
@@ -66,14 +66,13 @@ export class ProjectsService {
    * @param      {string}  contactId   CRM Contact contactid
    * @return     {any}     { ...Project Attributes }
    */
-  public async getProject(projectId: string, contactId: string) {
+  public async getProject(projectId: string) {
     try {
       const { records } = await this.crmService.get('dcp_projects', `
         $filter=
           dcp_dcp_project_dcp_projectapplicant_Project/
             any(o:
-              o/_dcp_applicant_customer_value eq '${contactId}'
```

Comment:
Wait, why remove the filtering here? 
Doesn't this method return projects assigned to the current user? 
without this filter, it seems it would return all projects with active applicants

Reply:
Oh nevermind, i see. this is for a single project. And we will set up a project access guard

Reply:
It would return a single project based on ID. If we want to enforce something, we should have a guard explicitly throw "False" which would generate an error.

Reply:
yeah we don't really even need any of the other filters there except for the id. we could also use the correct rest convention `dcp_projects/{:id}`

Reply:
Wait I don't get what you mean. except for which id? Also our endpoint does look like that: currently it is `projects/{:id}` ?